### PR TITLE
add CLON

### PR DESCRIPTION
**Pull request by clonefund**
add CLON

### CODE REVIEW

File: terra2/assetlist.json [hunk 1]
```diff
@@ -366,6 +366,33 @@
...
+          "denom": "CLON",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:terra164ssz60yvsxey0ku9mtcaegdeyxwzuwwqyrp238nvflwqve0pvxsra7fa2",
+      "name": "CLON",
+      "display": "CLON",
+      "symbol": "CLON",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/clon1.png"
```
Comment (JeremyParish69) [inline terra2/assetlist.json:388]: Can you please remove the extra padding from the image? The logo will appear tiny with all that padding

**Comment by clonefund** [comment]
think now its ok sir

File: terra2/assetlist.json [hunk 2]
```diff
@@ -366,6 +366,33 @@
         }
       ]
     },
+    {
+      "description": "ClonEliteFund invests in Cosmos ecosystem projects, focusing on scalability, security, and interoperability to maximize returns.",
```
Comment (JeremyParish69) [inline terra2/assetlist.json:370]: Please relocate this asset to the bottom of the list.

Comment (clonefund) [reply terra2/assetlist.json:370]: sir, is it really necessary, I see that the code layout in other entries is the same as mine, tell me where exactly I should move it or please do it if you can

Comment (JeremyParish69) [reply terra2/assetlist.json]: the layout of properties within the asset was fine, but new assets always go to the bottom of the list--you just inserted this somewhere in the middle of the list

**Pull request merged**